7_deploy_osh/roles/suse-build-images/tasks/main.yml

```diff
@@ -0,0 +1,88 @@
+---
+
+- name: Install system packages for build images
+  package:
+    name: "{{ suse_build_images_packages }}"
+    state: present
+  register: install_packages
+  until: install_packages is success
+  retries: 5
+  delay: 2
+  tags:
+    - install
+
+- name: Check if local docker registry server is running
+  shell: |
+    docker ps | grep {{ local_registry_name }}
+  register: registry_running_cmd_status
+  failed_when: false
+  tags:
+    - run
+
+- name: Start local docker registry if not running
+  shell: |
+    docker run -d -p {{ local_registry_port }}:{{ local_registry_port }} --name {{ local_registry_name }} --restart always registry:2
```

Comment:
This doesn't ensure that the registry named {{ local_registry_name }} runs on the expected port.
I am fine with that, as its for dev purposes only, but I'd prefer seeing a comment about it before the task.

Reply:
We are specifying local_registry_port whose default value is 5000. Currently same port is mapped on host and container side.
I am not sure what exactly you are expecting which is missing here.